toolchain/sem_ir/id_kind.h

```diff
@@ -141,11 +141,11 @@ using IdKind = TypeEnum<
     // From sem_ir/ids.h.
     InstId, AbsoluteInstId, DestInstId, MetaInstId, AnyRawId, ConstantId,
     EntityNameId, CompileTimeBindIndex, CallParamIndex, FacetTypeId, FunctionId,
-    ClassId, InterfaceId, AssociatedConstantId, ImplId, GenericId, SpecificId,
-    ImportIRId, ImportIRInstId, LocId, BoolValue, IntKind, NameId, NameScopeId,
-    InstBlockId, AbsoluteInstBlockId, DeclInstBlockId, LabelId, ExprRegionId,
-    StructTypeFieldsId, TypeId, TypeBlockId, ElementIndex, LibraryNameId,
-    FloatKind>;
+    ClassId, InterfaceId, AssociatedConstantId, ImplId, SpecificInterfaceId,
+    GenericId, SpecificId, ImportIRId, ImportIRInstId, LocId, BoolValue,
```

Comment:
This list is admittedly a mess (I kind of wonder if, for deltas, we should put one-per-line and disable clang-format if needed to get there), but I _think_ it's trying to keep an order similar to ids.h, meaning:

```suggestion
    ClassId, InterfaceId, AssociatedConstantId, ImplId, 
    GenericId, SpecificId, SpecificInterfaceId, ImportIRId, ImportIRInstId, LocId, BoolValue,
```

Reply:
Yeah, one per line would be much nicer. Done.

Matching orders is tough to maintain, but arbitrary ordering is no good - how about I sort these? Not sure if that's nice enough, but much easier to keep correct. See what you think?